pkg/trace/api/otlp.go

```diff
@@ -113,13 +112,13 @@ func (o *OTLPReceiver) Stop() {
 	o.wg.Wait()
 }
 
-// Export implements otlppb.TraceServiceServer
-func (o *OTLPReceiver) Export(ctx context.Context, in *otlppb.ExportTraceServiceRequest) (*otlppb.ExportTraceServiceResponse, error) {
+// Export implements otlpgrpc.TracesServer
```

Comment:
I usually do that when there is nothing requiring the interface in the same package, telling me that it's not implemented correctly. But in this case we register the server in the same file so there wouldn't be much point? How would the error be different if I did that?

Reply:
In this case the error would be the same, but I usually do this regardless just in case the code changes; here it's unlikely that we don't ever register the server so we can keep it like this